userspace/libsinsp/container_engine/docker_common.cpp

```diff
@@ -351,6 +355,13 @@ bool docker_async_source::parse_docker(std::string &container_id, sinsp_containe
 						}
 					}
 				}
+				// fix image digest for locally tagged images or multiple repo digests.
+				// Case 1: One repo digest with many tags.
+				// Case 2: Many repo digests with the same digest value.
+				if(container->m_imagedigest.empty() && img_root["RepoDigests"].size() >= 1 &&
```

Comment:
Why do we need `img_root["RepoDigests"].size() >= 1`?

Reply:
Oou yeah! We don't, let me get rid of it.